src/containers/shared/transactionUtils.ts

```diff
@@ -167,14 +203,14 @@ const zeroPad = (num, size, back = false) => {
   return s;
 };
 
-export const normalizeAmount = (amount, language = 'en-US') => {
-  const currency = amount.currency || 'XRP';
-  const value = amount.value || amount / XRP_BASE;
+export const normalizeAmount = (amount: IssuedCurrencyAmount | number, language = 'en-US') => {
+  const currency = typeof amount === 'object' ? amount.currency : 'XRP';
+  const value = typeof amount === 'object' ? amount.value : amount / XRP_BASE;
```

Comment:
It might be nice to use parentheses here to make it clear that the code will execute the right order between "(typeof amount === 'object') ? ..." or "(typeof amount === ('object' ? amount.currency : 'XRP'))"

Reply:
I like to structure my ternaries as 
```ts
const value = typeof amount === 'object'
    ? amount.value
    : amount / XRP_BASE;
```
Similar reason as using parenthesis, it improves readability.

Up to you what you wanna do here, as long as the linter is cool w/ it.

Reply:
`prettier` gets mad at me if I do either of those things, so I'm just going to leave it alone